src/validation.cpp

```diff
@@ -5066,6 +5066,8 @@ double GuessVerificationProgress(const ChainTxData& data, const CBlockIndex *pin
         fTxTotal = pindex->nChainTx + (nNow - pindex->GetBlockTime()) * data.dTxRate;
     }
 
+    if (pindex->nChainTx > fTxTotal) return 1;
+
     return pindex->nChainTx / fTxTotal;
```

Comment:
please return `1.0` instead of `1` (more clear that it's a double/fraction)
or maybe collapse it into one statement
```
return std::min(pindex->nChainTx / fTxTotal, 1.0);
```

Reply:
that's fine with me too; I just think the `return 1; ` looks strange in a function that returns double

Reply:
Corrected, thanks!